### PR TITLE
completed javascript-if

### PR DESCRIPTION
**Pull request by jdfenlason**
Screen Shot 2021-03-17 at 6 53 22 PM

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  function categorizeAcidity(ph) {
+    if(ph === 7) {
+      return 'neutral'
+    } if (ph < 7 && ph > 0){
+     return 'acid'
+    } if(ph > 8 && ph < 14) {
+    return 'base'
+  } else {
+    return 'invalid ph level'
+  }
```
Comment (codingcodymiller) [inline javascript-if/main.js:92]: Careful of your indentation, it starts to get pretty funky in this section.

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+function introduceWarnerBro(name) {
+   if(name=== 'yakko' || name=== 'wakko') {
+ return "We're the warner brothers"
+   } if(name === 'dot') {
+    return "I'm cute~"
+  }
+  else {
+    return "Goodnight everybody"
+}
```
Comment (codingcodymiller) [inline javascript-if/main.js:103]: Same here, fix up this indentation

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      return false
+    }
+  };
+  function startsWithJ(string) {
+    if(string[0] === 'J') {
+      return true
+    } else {
+        return false
+      }
+    };
```
Comment (codingcodymiller) [inline javascript-if/main.js:62]: Looks like your indentation is a bit funky here too, please get this indentation to be consistent and follow proper code formatting rules.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-if/main.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+var shia = {
+  name: "Shia Lebouf",
+  age: 34,
+};
+
+function isOldEnoughToDrink(age) {
+  if ((age) => 21) {
```
Comment (codingcodymiller) [inline javascript-if/main.js:16]: You have the characters backwards here. With "less than or equal to" and "greater than or equal to", the angle bracket comes first. with the equal sign first, you are using an entirely different operator we haven't talked about yet. This should be `>=`. You have this off in a couple other definitions too, double check your others while you fix this. Also, make sure you test your functions again to see if they are working as expected still

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! I left a note for you to address

**Comment by jdfenlason** [comment]
Screen Shot 2021-03-18 at 8 23 28 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**